specification/provisioningservices/resource-manager/Microsoft.Devices/2017-11-15/examples/DPSDelete.json

```diff
@@ -0,0 +1,14 @@
+{
+    "parameters": {
+        "provisioningServiceName": "myFirstProvisioningService",
+        "resourceGroupName": "myResourceGroup",
+        "api-version": "2017-11-15",
+        "subscriptionId": "91d12660-3dec-467a-be2a-213b5544ddc0"
+    },
+    "responses": {
+      "200": {},
```

Comment:
>  "200": {}, [](start = 4, length = 12)

when do we return 200? If there is no content, it should be 204.

Reply:
This is returned as a response to the status polling request for the delete operation. The body contains the resource representation that indicates a transitional provisioning state.